### PR TITLE
SONAR-8337 add propertes for es logs

### PR DESCRIPTION
**Pull request by sns-seb**
based on #1394

### CODE REVIEW

File: server/sonar-process/src/main/java/org/sonar/process/LogbackHelper.java [hunk 1]
```diff
@@ -254,18 +259,50 @@ public boolean isAllLogsToConsoleEnabled(Props props) {
...
    * @throws IllegalArgumentException if the value of the specified property is not one of {@link #ALLOWED_ROOT_LOG_LEVELS}
    */
-  public Level configureRootLogLevel(Props props, String propertyKey) {
-    Level newLevel = Level.toLevel(props.value(propertyKey, Level.INFO.toString()), Level.INFO);
-    return configureRootLogLevel(newLevel);
+  public Level configureRootLogLevel(Props props, String... propertyKeys) {
+    return configureLoggerLogLevel(getRootContext().getLogger(ROOT_LOGGER_NAME), props, propertyKeys);
+  }
+
+  private Level configureLoggerLogLevel(Logger logger, Props props, String... propertyKeys) {
```
Comment (SonarTech) [inline server/sonar-process/src/main/java/org/sonar/process/LogbackHelper.java]: MAJOR Make "configureLoggerLogLevel" a "static" method. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2325)

File: server/sonar-process/src/main/java/org/sonar/process/LogbackHelper.java [hunk 2]
```diff
@@ -254,18 +259,50 @@ public boolean isAllLogsToConsoleEnabled(Props props) {
...
+    if (!isAllowed(level)) {
+      throw new IllegalArgumentException(String.format("log level %s in property %s is not a supported value (allowed levels are %s)",
+        level, propertyKey, Arrays.toString(ALLOWED_ROOT_LOG_LEVELS)));
+    }
+    return level;
+  }
+
+  private static boolean isAllowed(Level level) {
+    for (Level allowedRootLogLevel : ALLOWED_ROOT_LOG_LEVELS) {
+      if (level == allowedRootLogLevel) {
```
Comment (SonarTech) [inline server/sonar-process/src/main/java/org/sonar/process/LogbackHelper.java]: MAJOR Change this comparison to use the equals method. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1698)

File: sonar-db/src/main/java/org/sonar/db/DefaultDatabase.java [hunk 3]
```diff
@@ -93,7 +97,7 @@ private void initDataSource() throws Exception {
     datasource = new ProfiledDataSource(basicDataSource, NullConnectionInterceptor.INSTANCE);
     datasource.setConnectionInitSqls(dialect.getConnectionInitStatements());
     datasource.setValidationQuery(dialect.getValidationQuery());
-    enableSqlLogging(datasource, "TRACE".equals(settings.getString("sonar.log.level")));
+    enableSqlLogging(datasource, logbackHelper.getLoggerLevel("sql") == Level.TRACE);
```
Comment (SonarTech) [inline sonar-db/src/main/java/org/sonar/db/DefaultDatabase.java]: MAJOR Change this comparison to use the equals method. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1698)

**Comment by SonarTech** [comment]
SonarQube analysis reported 3 issues
* MAJOR 3 major

Watch the comments in this conversation to review them.

File: sonar-application/src/main/assembly/conf/sonar.properties [hunk 4]
```diff
@@ -235,6 +235,12 @@
 #sonar.log.level.web.sql=TRACE
 #sonar.log.level.ce.sql=TRACE
 
+# Es client logs in the Web and CE processes can be individually controlled with their respective properties
```
Comment (simonbrandhof) [inline sonar-application/src/main/assembly/conf/sonar.properties]: I suggest to replace "es" by "Elasticsearch" in comment.

**Pull request merged**